MaterialLib/PorousMedium/UnsaturatedProperty/CapillaryPressure/VanGenuchtenCapillaryPressureSaturation.cpp

```diff
@@ -51,10 +52,8 @@ double VanGenuchtenCapillaryPressureSaturation::getCapillaryPressure(
 double VanGenuchtenCapillaryPressureSaturation::getSaturation(
     const double capillary_pressure) const
 {
-    const double pc =
-        (capillary_pressure < 0.0) ? _minor_offset : capillary_pressure;
-    double Se = std::pow(pc / _pb, 1.0 / (1.0 - _m)) + 1.0;
-    Se = std::pow(Se, -_m);
+    const double pc = std::max(_minor_offset, capillary_pressure);
```

Comment:
The previous expression and the new one are not equal. For instance if `capillary_pressure = 0.1` and `_minor_offset = 0.5` the previous expression set `pc` to `0.1` while the new expression results in `pc = 0.5`. Is this intended?

Reply:
Intended. This moves the cutoff to correct point; the pc value is set to "minor_offset" not only for non-positive values, but for pc < minor_offset.
With capillary_pressure(x) = x, minor_offset = 0.1, the pc(x = 0.05) = 0.05 in the previous implementation, now it is does not have this "negative tooth" between x = 0 and 0.1.